src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-list/multi-cluster-list.component.scss

```diff
@@ -0,0 +1,7 @@
+@use '../../../../../styles/vendor/variables' as vv;
+
+.fa-wrench {
+  color: vv.$info;
+  font-size: 6em;
+  margin-top: 200px;
+}
```

Comment:
instead of this why not `[ngClass]="[icons.large, icons.wrench]"`.
also `margin-top: 200px` is a weird way of styling and it can cause issues with responsiveness and downstream branding. Unless you want to deal with those I'd suggest finding a better way to deal with it. (mostly reusing bootstrap classes like `justify-content-center` and `align-items-center` for centering the icon if that was your intention)

Reply:
removed `margin-top:200px`, but we'l need to keep the font size as icons.large is still not the required size here.